src/test/java/org/bcnjug/jbcn/api/paper/PaperControllerCreateTest.java

```diff
@@ -0,0 +1,98 @@
+package org.bcnjug.jbcn.api.paper;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.bcnjug.jbcn.api.papers.Paper;
+import org.bcnjug.jbcn.api.papers.PapersRepository;
+import org.bcnjug.jbcn.api.sec.Roles;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.security.test.context.support.WithMockUser;
+import org.springframework.test.web.reactive.server.WebTestClient;
+import org.springframework.web.reactive.function.BodyInserters;
+import reactor.core.publisher.Mono;
+import reactor.test.StepVerifier;
+
+import java.nio.charset.StandardCharsets;
+import java.time.ZoneOffset;
+import java.util.Map;
+import java.util.UUID;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+@AutoConfigureWebTestClient
+class PaperControllerCreateTest {
```

Comment:
Good idea to have a test per method, usually Controller test get too long, but then maybe we should keep names simpler like "CreatePaperTest". We could place this tests in a "e2e" or "acceptance"  package also to show this are not unitary.